### PR TITLE
Adding program to find pangram

### PR DESCRIPTION
**Pull request by Ihjass**
A pangram is a sentence using every letter of the alphabet at least once.
pangram-output

### CODE REVIEW

**Review by Panquesito7 — DISMISSED** [review]
Thanks! 🚀

File: src/strings/pangram.jl [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+end
+
+```
+
+# References:
+(https://exercism.org/tracks/julia/exercises/pangram)
+
+```
+
+# Contributed by:- [Ihjass](https://github.com/Ihjass)
```
Comment (Panquesito7) [inline src/strings/pangram.jl:42]:
```suggestion
# Contributed by:- [Ihjass Thasbekha](https://github.com/Ihjass)
```

**Review by Panquesito7 — APPROVED** [review]

**Review by Panquesito7 — CHANGES_REQUESTED** [review]
I think you need to add your program in the tests folder as seen in other files https://github.com/TheAlgorithms/Julia/tree/main/test

**Review by uncomfyhalomacro — DISMISSED** [review]
Thanks for the contribution! :D

**Comment by uncomfyhalomacro** [comment]
@Ihjass pls do add test for it to pass the CI. thank you!!

**Review by Panquesito7 — CHANGES_REQUESTED** [review]
Looks like all the other [**checks**](https://github.com/TheAlgorithms/Julia/pull/100/checks) are failing 😅 could you please fix that?
Let us know if you need any help here or in our [**Discord server**](https://discord.gg/c7MnfGFGa6). 😃

**Review by uncomfyhalomacro — DISMISSED** [review]

**Review by uncomfyhalomacro — APPROVED** [review]

**Review by Panquesito7 — COMMENTED** [review]
LGTM! Thanks for fixing that, @uncomfyhalomacro! 🚀

**Pull request merged**